pom.xml

```diff
@@ -171,6 +181,57 @@
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-install-plugin</artifactId>
 				<version>3.1.1</version>
+				<executions>
+					<!-- Install main JAR -->
```

Comment:
Can we remove this unnecessary comment?

Reply:
Addressed in [e5a7f84a664b8e666a8cd00b5e15224a6946da2d](https://github.com/ls1intum/Ares2/pull/25/commits/e5a7f84a664b8e666a8cd00b5e15224a6946da2d)